nominal/core/run.py

```diff
@@ -144,6 +144,43 @@ def archive(self) -> None:
         """
         self._clients.run.archive_run(self._clients.auth_header, self.rid)
 
+    def remove_ref_names(self, ref_names_to_remove: Iterable[str]) -> Self:
```

Comment:
nit: don't love this func name, but I get what you were going for.

what if we put it all in one method and had:

```py
def remove_dataset(self, *, rids: Iterable[str] = (), ref_names: Iterable[str] = ())
```
and you could call it with `rids=[r1, r2]` or `ref_names=[ref1, ref2]`?

or maybe just `remove_datasets_by_rid()` and `remove_datasets_by_refname()`?

Reply:
And I think either way I probably would just remove the singular function, doesn't seem like an "important" enough task to have a ton of functions for it